utils/details.js

```diff
@@ -135,7 +164,22 @@ class ActionDetails {
     const actions = await this.extractActionFromConfig()
 
     for await (const action of actions) {
-      const {owner, repo, version, position} = action
+      const {owner, repo, version, position, registry} = action
+
+      if (registry) {
+        this.postReviewComment(
+          `## :whale: Docker container image detected
+
+**This will add the Docker container image to the allow list!**
+
+${repo ? `:link: https://${registry}/${repo}${version ? `/${version}` : ''}` : `\`${registry}\``}
```

Comment:
Could you explain what is going on here please?

I'm having some difficulties reading the nested ternary and template literals usage.  At first glance, I assume there is an error as

```javascript
`:link: https://${registry}/${repo}${version
```

at first glance seems like it would be:

```javascript
`:link: https://${registry}/${repo}${version}`
```

Reply:
This is to support entries like:

```yaml
  - docker://*.example.com/*
```

So only comment with `*.example.com/*` rather than creating a clickable link.